web_server/users/static/users/css/mandala.css

```diff
@@ -0,0 +1,167 @@
+/*
+
+Copyright (c) 2020 - YIFEI BAI - https://codepen.io/soju22/pen/gEYJoO
```

Comment:
This copyright notice is confusing. You have stated that you have the copyright, but then link to codepen.io, then the license notice is MIT. Our entire project is by default MIT licensed, and I dont think copyright is something we can claim off a codepen. I would remove the license notice, and replace it with an author statement (e.g. @Author: Yifei Bai) and then provide the codepen link. To be polite, it might also be of value to also state the author of the original codepen.